source/core/types/ut_suite_item.tpb

```diff
@@ -12,6 +12,7 @@ create or replace type body ut_suite_item as
     self.path := nvl(lower(trim(a_path)), self.object_name);
     self.rollback_type := a_rollback_type;
     self.ignore_flag := ut_utils.boolean_to_int(a_ignore_flag);
+    self.results_count := ut_results_counter();
```

Comment:
the counter on suite level aggregates results from items within the suite. It cannot be reset as we would loose the data.